datacompy/fugue.py

```diff
@@ -26,13 +26,35 @@
 import pandas as pd
 import pyarrow as pa
 from fugue import AnyDataFrame
+from ordered_set import OrderedSet
 
 from .core import Compare, render
 
 LOG = logging.getLogger(__name__)
 HASH_COL = "__datacompy__hash__"
 
 
+def unq_columns(df1: AnyDataFrame, df2: AnyDataFrame):
+    """Get columns that are unique to df1
+
+    Parameters
+    ----------
+    df1 : ``AnyDataFrame``
+        First dataframe to check
+
+    df2 : ``AnyDataFrame``
+        Second dataframe to check
+
+    Returns
+    -------
+    OrderedSet
+        Set of columns that are unique to df1
+    """
+    tdf1 = fa.as_fugue_df(df1)
+    tdf2 = fa.as_fugue_df(df2)
+    return OrderedSet(tdf1.columns) - OrderedSet(tdf2.columns)
```

Comment:
actually, we shouldn't convert them to fugue df in this case.
https://fugue.readthedocs.io/en/latest/top_api.html#fugue.api.get_column_names

```python
columns1 = fa.get_column_names(df1)
...
```

This can bypass any unnecessary data conversion.

Reply:
Perfect!